lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -328,6 +328,7 @@ def _section_data_download(course, access):
         'list_report_downloads_url': reverse('list_report_downloads', kwargs={'course_id': course_key.to_deprecated_string()}),
         'calculate_grades_csv_url': reverse('calculate_grades_csv', kwargs={'course_id': course_key.to_deprecated_string()}),
         'get_student_responses_url': reverse('get_student_responses', kwargs={'course_id': course_key.to_deprecated_string()}),
+        'get_student_forums_usage_url': reverse('get_student_forums_usage', kwargs={'course_id': course_key.to_deprecated_string()}),
```

Comment:
New code should _almost_ *never* call `to_deprecated_string` or `from_deprecated_string`.

I know the other code here still follows this pattern, but that's because we're behind upstream.  They have already converted their legacy invocations [1]; we should do the same for our new ones.

[1] https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor/views/instructor_dashboard.py#L161